app/Http/Controllers/API/SuggestController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Models\Building;
+use App\Models\Architect;
+
+class SuggestController extends Controller
```

Comment:
```suggestion
class SearchSuggestionController extends Controller
```

(The controller name should refer to resource, not action)